pwndbg/enhance.py

```diff
@@ -30,8 +31,25 @@ def good_instr(i) -> bool:
     return not any(bad in i for bad in bad_instrs)
 
 
+def format_small_int(value: int) -> str:
+    if value < 10:
+        return "%d" % value
+    else:
+        return "%#x" % int(value & pwndbg.gdblib.arch.ptrmask)
```

Comment:
`hex(value & pwndbg.gdblib.arch.ptrmask)` would be better here.

Reply:
Why do we want to have 0x for =10 but not for <10?

Reply:
I was mostly doing this for personal preference, and seeing that other parts of the code did the same:

https://github.com/pwndbg/pwndbg/blob/45665e823c7736ff25006d2b77426df7998dc00c/pwndbg/enhance.py#L111-L114

There another location that keeps it as decimal when the value is < 256:
https://github.com/pwndbg/pwndbg/blob/45665e823c7736ff25006d2b77426df7998dc00c/pwndbg/color/memory.py#L56-L59

Perhaps we make this a global configurable setting - i.e. `max-decimal-string-number` or something like that.

Reply:
I like the configuration idea, but let's focus on getting this PR merged and implement any additional features in future PRs.